chapter_02_unittest.asciidoc

```diff
@@ -229,6 +231,8 @@ class NewVisitorTest(unittest.TestCase):  # <1>
         self.browser.quit()
 
     def test_can_start_a_todo_list(self):  # <2>
+        // DAVID: Pedantry: should we be consistent with whether
+        // comments end with a full stop?
```

Comment:
reminder to avoid suggesting changes directly into code unless absolutely necessary.  will re add below.
```suggestion
```